typescript/sdk/src/metadata/chainMetadataTypes.ts

```diff
@@ -67,6 +67,29 @@ export const RpcUrlSchema = z.object({
 
 export type RpcUrl = z.infer<typeof RpcUrlSchema>;
 
+export const BlockExplorerSchema = z.object({
```

Comment:
Broke this out of the bigger schema so I can use it to validate explorer defs individually later. No field changes or anything here.